bundles/AdminBundle/Helper/GridHelperService.php

```diff
@@ -683,6 +683,22 @@ public function prepareAssetListingForGrid($allParams, $adminUser)
                                                  )';
         }
 
+        //filtering for tags
+        if (!empty($allParams['tagIds'])) {
+            $tagIds = $allParams['tagIds'];
+            foreach ($tagIds as $tagId) {
+                if (($allParams['considerChildTags'] ?? 'false') === 'true') {
```

Comment:
can we please change the submit value ?
see https://docs.sencha.com/extjs/6.2.0/classic/Ext.form.field.Checkbox.html#cfg-submitValue